### ISSUE TITLE
Adding EiffelActivityStartedEvent as a legal target in links.context of EiffelTestSuiteStartedEvent

### ISSUE THREAD

**Issue by ebram96** [eiffel-community/eiffel#332]
### Description

 I think it'd be good in chronological sequence to also allow using `EiffelActivityStartedEvent` as a legal target for links.context of `EiffelTestSuiteStartedEvent`.

### Motivation

Since a test suite will never start unless its parent activity starts and I think it's better to think of it as:
The test suite x started when that activity started.
Instead of:
The test suite x started when that activity was triggered.
Since triggering an activity doesn't actually start any suites, but, starting an activity may start suites.

### Exemplification

Maybe when visualizing chronological order it'd be better to see `ActivityStarted -> TestSuiteStarted` instead of `ActivityTriggered -> TestSuiteStarted`.

### Benefits

More convenient flow.

### Possible Drawbacks

I'm not sure.

**Comment by e-backmark-ericsson** [comment]
@ebram96 , we'd like all issues to use the issue template that we've prepared, to make sure all aspects of an issue are considered when describing it. Could you please update your issue and apply the issue template (the one that comes up when clicking "New issue")?

**Comment by ebram96** [comment]
> @ebram96 , we'd like all issues to use the issue template that we've prepared, to make sure all aspects of an issue are considered when describing it. Could you please update your issue and apply the issue template (the one that comes up when clicking "New issue")?

I updated it.

**Comment by e-backmark-ericsson** [comment]
Thanks for the update @ebram96 

Regarding your request, the CONTEXT link describes "in what context" a certain event occurred. Or as it is stated in the documentation: "Identifies the activity or test suite of which this event constitutes a part.". All event types can have CONTEXT links. They don't specify what was the cause of something happening and it doesn't specify after what other event this event occurred. You can read more about activity linking here: https://github.com/eiffel-community/eiffel/blob/master/eiffel-syntax-and-usage/activity-linking.md

From an event consumer perspective, I'd say it is probably more relevant to ask "within what activity/pipeline/step was this test suite executed" than "when was the activity, that this test suite is part of, started".

So, clarifying a bit more, the CONTEXT link doesn't describe "The test suite x started _when_ that activity was triggered.", but rather "The test suite x started within the context of that activity."

Does this make sense?

**Comment by ebram96** [comment]
I see, that makes sense. Thanks for elaborating..
Closing.
